pkg/testutil/api.go

```diff
@@ -0,0 +1,84 @@
+package testutil
```

Comment:
This is the new `testutil` package file that contains the shared test data now consumed by all tests.